src/Illuminate/Testing/TestResponse.php

```diff
@@ -809,13 +686,15 @@ public function assertJsonMissingValidationErrors($keys = null, $responseKey = '
      * Validate and return the decoded response JSON.
      *
      * @param  string|null  $key
-     * @return mixed
+     * @return \Illuminate\Testing\TestJson
      *
      * @throws \Throwable
      */
     public function decodeResponseJson($key = null)
     {
```

Comment:
$key is no longer used in this method at all. This appears to be a breaking change? Should it still be used? More info?

Reply:
Missed that, fixed now.

Although I still kinda unsure if we should stick with `decodeResponseJson()` or rename it to something else. Out of idea right now.

Reply:
I see you removed the parameter. Nobody would have any use for this parameter anymore? It is never helpful?

Reply:
The same usage is available via `TestResponse::json($key)`. If we still want to remain the parameter then the possible return type for `decodeJsonResponse()` will be `Illuminate\Testing\TestJson` and  `mixed` (or `null` if `$key` is set)